Makefile.kokkos

```diff
@@ -1198,6 +1217,21 @@ ifeq ($(KOKKOS_INTERNAL_USE_HIP), 1)
   endif
 endif
 
+# Figure out the architecture flag for ROCm.
```

Comment:
may wanna fix the comment :-)

Reply:
ooops :slightly_smiling_face: Fixed in 479bdfa